opendbc/car/tesla/carstate.py

```diff
@@ -44,7 +45,8 @@ def update(self, can_parsers) -> structs.CarState:
     ret.steeringTorque = -epas_status["EPAS3S_torsionBarTorque"]
 
     # This matches stock logic, but with halved minimum frames (0.25-0.3s)
-    ret.steeringPressed = self.update_steering_pressed(abs(ret.steeringTorque) > STEER_THRESHOLD, 15)
+    ret.steeringPressed = self.update_steering_pressed(abs(ret.steeringTorque) > STEER_THRESHOLD, 15) or \
+                          self.steering_override
```

Comment:
Faster detection of slightly higher torque to avoid "steering limited" when using torque blending:

```suggestion
    ret.steeringPressed = (self.update_steering_pressed(abs(ret.steeringTorque) > STEER_THRESHOLD, 15) or
        self.steering_override or
        self.update_steering_pressed(abs(ret.steeringTorque) > 1.0, 5))
```